app/src/main/java/org/stacks/app/ui/account/AccountViewModel.kt

```diff
@@ -0,0 +1,33 @@
+package org.stacks.app.ui.account
+
+import androidx.hilt.lifecycle.ViewModelInject
+import kotlinx.coroutines.channels.BroadcastChannel
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+import org.stacks.app.data.IdentityModel
+import org.stacks.app.data.interfaces.IdentityRepository
+import org.stacks.app.ui.BaseViewModel
+
+class AccountViewModel
+@ViewModelInject constructor(
+    identityRepository: IdentityRepository
+) : BaseViewModel() {
+
+    // Outputs
+    private val identities = BroadcastChannel<List<IdentityModel>>(1)
+
+    init {
+        identityRepository
+            .observe()
+            .onEach {
+                identities.send(it)
+            }
+            .launchIn(ioScope)
```

Comment:
Do we need to double-check if the user is authenticated here, and take him/her out of this activity?